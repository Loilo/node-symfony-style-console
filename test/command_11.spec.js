

const wrapper = require('./wrapper')

test('11 Ensure long words are properly wrapped in blocks', wrapper(io => {
	io.block(
		'Lopadotemachoselachogaleokranioleipsanodrimhypotrimmatosilphioparaomelitokatakechymenokichlepikossyphophattoperisteralektryonoptekephalliokigklopeleiolagoiosiraiobaphetraganopterygon',
		'CUSTOM',
		'fg=white;bg=blue',
		' ยง ',
		false
	)
}))
