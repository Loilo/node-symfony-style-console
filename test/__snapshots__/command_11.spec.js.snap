// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`11 Ensure long words are properly wrapped in blocks 1`] = `
"ยง [CUSTOM] Lopadotemachoselachogaleokranioleipsanodrimhypotrimmatosilphioparaom
 ยง          elitokatakechymenokichlepikossyphophattoperisteralektryonoptekephall
 ยง          iokigklopeleiolagoiosiraiobaphetraganopterygon"
`;
